kiva/ui
# feat: component renamed and route updated VUE-1005

**Steps to test:**

1. Go to [http://localhost:8888/lend/filter-alpha](http://localhost:8888/lend/filter-alpha)
2. You should see the **Loan Search Page** correctly
